{
  "ID": "pyup.io-39522",
  "Advisory": "Django 3.0.12 fixes a security issue with severity \"low\" in 3.0.11 (CVE-2021-3281).",
  "Cve": "CVE-2021-3281",
  "Specs": [
    "==3.0.11"
  ],
  "v": "==3.0.11"
}
